src/Sarif.Multitool/Program.cs

```diff
@@ -0,0 +1,28 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.IO;
+using CommandLine;
+using Microsoft.CodeAnalysis.Sarif.Converters;
+using Microsoft.CodeAnalysis.Sarif.Writers;
+
+namespace Microsoft.CodeAnalysis.Sarif.ConvertToSarif
+{
+    internal static class Program
+    {
+        /// <summary>The entry point for the SARIF file format conversion utility.</summary>
+        /// <param name="args">Arguments passed in from the tool's command line.</param>
+        /// <returns>0 on success; nonzero on failure.</returns>
+        public static int Main(string[] args)
+        {
+            return Parser.Default.ParseArguments<
+                ConvertOptions, 
+                RewriteOptions>(args)
+              .MapResult(
```

Comment:
>MapResult [](start = 15, length = 9)

I believe, because of the particular form of the lambdas, you can express this more simply as:
```
.MapResult(
    ConvertCommand.Run,
    RewriteCommand.Run,
    errs => 1
```

Reply:
no, 'parameterization cannot be inferred from usage'